src/main/java/ee/tuleva/onboarding/payment/provider/montonio/MontonioPaymentChannelConfiguration.java

```diff
@@ -14,16 +14,16 @@
 @Configuration
 @RequiredArgsConstructor
 @ConfigurationProperties("payment-provider")
-class PaymentProviderConfiguration {
+public class MontonioPaymentChannelConfiguration {
```

Comment:
Does it need to be public?

Reply:
[Same comment as with PaymentInternalReferenceService](https://github.com/TulevaEE/onboarding-service/pull/899#discussion_r1665596326)